assets/AnVIL_style/style.css

```diff
@@ -303,6 +303,19 @@ div.reflection{
   background-color: #e8ebee;
   background-image: url("../assets/box_images/thinking_face.png");
 }
+div.example{
```

Comment:
@avahoffman can we delete div.example? or did you want it for something?

Reply:
yes let's delete it!